### DISCUSSION TITLE
Printer kill

### DISCUSSION THREAD

**Discussion by ender3v2** [Jyers/Marlin#207] [Q&A] [answered]
What is that about? 
![20210308_025350](https://user-images.githubusercontent.com/80234914/110270388-faec3100-7fc5-11eb-9bc1-f0014da8273b.jpg)

**Comment by darco415** [comment]
already being discussed...... https://github.com/Jyers/Marlin/discussions/192

**Reply by Xploder** [reply]
Nah this is a different error, not probing failed

**Comment by ender3v2** [comment]
Yep. It Comes when the heat is 245/230 it have to cool down before start to print. It doesnt. It starts the print OR this message comes

**Comment by ender3v2** [comment]
No it doesnt i See on the picture, this was another failure

**Comment by Jyers** [comment, marked as answer]
This is just a part of marlin's safety checks. You must have somehow triggered it. Nothing really to do with this particular branch of marlin, I added a popup so the user knows what happened, otherwise the printer would just stop working and you would have no explanation.
